syndesis/plugins/org.fusesource.ide.syndesis.extension.core/src/org/fusesource/ide/syndesis/extensions/core/internal/SyndesisExtensionsCoreActivator.java

```diff
@@ -0,0 +1,84 @@
+/*******************************************************************************
+ * Copyright (c) 2017 Red Hat, Inc.
+ * Distributed under license by Red Hat, Inc. All rights reserved.
+ * This program is made available under the terms of the
+ * Eclipse Public License v1.0 which accompanies this distribution,
+ * and is available at http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     Red Hat, Inc. - initial API and implementation
+ ******************************************************************************/
+
+package org.fusesource.ide.syndesis.extensions.core.internal;
+
+import org.eclipse.core.runtime.Platform;
+import org.jboss.tools.foundation.core.plugin.BaseCorePlugin;
+import org.jboss.tools.foundation.core.plugin.log.IPluginLog;
+import org.jboss.tools.foundation.core.plugin.log.StatusFactory;
+import org.osgi.framework.BundleContext;
+
+
+/**
+ * @author lhein
+ */
+public class SyndesisExtensionsCoreActivator extends BaseCorePlugin {
+	public static final String PLUGIN_ID = "org.fusesource.ide.syndesis.extension.core";
+
+	private static SyndesisExtensionsCoreActivator instance;
+
+	/**
+	 * default constructor
+	 */
+	public SyndesisExtensionsCoreActivator() {
+		instance = this;
+	}
+
+	/**
+	 * returns the instance
+	 * 
+	 * @return
+	 */
+	public static SyndesisExtensionsCoreActivator getDefault() {
+		return instance;
+	}
+	
+	public static BundleContext getBundleContext() {
```

Comment:
I think that getBundle would make more sense as the only place it is called, there is getBundleContext().getBundle() which is called.